rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1203,25 +1315,36 @@ def _prepare_sparse_dense_layers(
                 )
 
     def _prepare_input_layers(self, name: Text) -> None:
-        self._tf_layers[f"sparse_input_dropout.{name}"] = layers.SparseDropout(
-            rate=self.config[DROP_RATE]
-        )
-        self._tf_layers[f"dense_input_dropout.{name}"] = tf.keras.layers.Dropout(
-            rate=self.config[DROP_RATE]
-        )
-        self._prepare_sparse_dense_layers(
-            self.data_signature[f"{name}_features"],
-            name,
-            self.config[REGULARIZATION_CONSTANT],
-            self.config[DENSE_DIMENSION][name],
-        )
-        self._tf_layers[f"ffnn.{name}"] = layers.Ffnn(
+        self._tf_layers[f"dense_layer.{name}"] = layers.Ffnn(
```

Comment:
sorry, I'd keep `ffnn` here. I only meant to rename concat layer
```suggestion
        self._tf_layers[f"ffnn.{name}"] = layers.Ffnn(
```